### PR TITLE
test: add connect to Portfolio

### PR DESCRIPTION
**Pull request by chrisleewilcox**
## **Description**
Automate scenario for connecting wallet to Portfolio by tapping on Portfolio button on Wallet view.

## **Related issues**

Related: https://github.com/MetaMask/mobile-planning/issues/1728

## **Manual testing steps**

1. Go to this page...
2.
3.

## **Pre-merge author checklist**

- [ ] I’ve followed [MetaMask Contributor Docs](https://github.com/MetaMask/contributor-docs) and [MetaMask Mobile Coding Standards](https://github.com/MetaMask/metamask-mobile/blob/main/.github/guidelines/CODING_GUIDELINES.md).
- [ ] I've completed the PR template to the best of my ability
- [ ] I’ve included tests if applicable
- [ ] I’ve documented my code using [JSDoc](https://jsdoc.app/) format if applicable
- [ ] I’ve applied the right labels on the PR (see [labeling guidelines](https://github.com/MetaMask/metamask-mobile/blob/main/.github/guidelines/LABELING_GUIDELINES.md)). Not required for external contributors.

## **Pre-merge reviewer checklist**

- [ ] I've manually tested the PR (e.g. pull and build branch, run the app, test code being changed).
- [ ] I confirm that this PR addresses all acceptance criteria described in the ticket it closes and includes the necessary testing evidence such as recordings and or screenshots.

### CODE REVIEW

File: e2e/specs/wallet/portfolio-connect-account.spec.js [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+  afterAll(async () => {
+    await stopFixtureServer(fixtureServer);
+  });
+
+  it('should connect wallet account to portfolio', async () => {
+    await loginToApp();
+    await WalletView.isVisible();
+    if (device.getPlatform() === 'android') {
+      /* closing all browser tabs before tapping Portfolio to interact with webview */
+      //   await TabBarComponent.tapBrowser();
```
Comment (sethkfman) [inline e2e/specs/wallet/portfolio-connect-account.spec.js:41]: Remove commented code

**Comment by SamuelSalas** [comment]
Regression test run: https://app.bitrise.io/app/be69d4368ee7e86d/pipelines/30a5abc2-fb48-4e25-937c-cccc647e018a
Smoke test run: https://app.bitrise.io/app/be69d4368ee7e86d/pipelines/c5e4593a-4174-4bdd-870d-75bd1c8c1e53

File: e2e/specs/wallet/portfolio-connect-account.spec.js (hunk 1 above)
Comment (SamuelSalas) [reply e2e/specs/wallet/portfolio-connect-account.spec.js]: Done

File: app/components/UI/Tokens/index.tsx [hunk 2]
```diff
@@ -635,7 +635,7 @@ const Tokens: React.FC<TokensI> = ({ tokens }) => {
           style={styles.buyButton}
           onPress={onOpenPortfolio}
           label={strings('asset_overview.portfolio_button')}
-          {...generateTestId(Platform, PORTFOLIO_BUTTON)}
+          testID={PORTFOLIO_BUTTON}
```
Comment (cortisiko) [inline app/components/UI/Tokens/index.tsx:638]: can we put this in a selector object?

**Review by cortisiko — CHANGES_REQUESTED** [review]

File: app/components/UI/Tokens/index.tsx (hunk 2 above)
Comment (SamuelSalas) [reply app/components/UI/Tokens/index.tsx]: Done

**Review by cortisiko — APPROVED** [review]
🌮 🌮 🌮

**Comment by chrisleewilcox** [comment]
@SamuelSalas looks good!

**Pull request merged**